### ISSUE TITLE
Error - Could not export the wallet account - purser-trezor

### ISSUE THREAD

**Issue by fuguefoundation** [JoinColony/purser#198]
`ERROR Error: Uncaught (in promise): Error: Could not export the wallet account, check the values you are sending to the Trezor service: type (xpubkey), requiredFirmware (1.4.0), path ([2147483692,2147483708,2147483648,0]), session not found`

I get the above error when calling `trezor.open()`. Trezor Bridge is running and I can see the session and connected device in the browser when I open up `http://127.0.0.1:21325/status/`. When the dapp loads, there is a pop up window prompting me for my pin, but the error appears immediately after I put in the pin. I have uninstalled the deprecated Trezor Chrome Extension so it does not conflict with Trezor Bridge. Any insight into this?

* Angular project
* Trezor firmware 1.6.3
* Node 10.12.0
* MacOS Mojave 10.14.2

**Comment by rdig** [comment]
> `session not found`

This points to a connection issue between the Trezor Bridge and the OS

Also, as I mentioned in the other issue _(#197)_ `node` @ `10` isn't supported yet. Most likely this is not the issue, but it's something to keep in mind.

I'll look into this, although it might be a bit tricky to track down...

**Comment by fuguefoundation** [comment]
`"@colony/purser-trezor": "^1.2.2"`

Thanks for your help on this!

**Comment by rdig** [comment]
It seems that I can't replicate this.

My initial suspicion was that it's something to do with the `node` version, but it works just fine on both node `8` and node `10`.

I've set up a repo that tests this: [rdig/purser-node10](https://github.com/rdig/purser-node10)

If you have some time, can you please clone the repo and test it locally? Thanks :+1:

**Comment by fuguefoundation** [comment]
Worked well! Guess I will continue to dig into the issue within the context of my project.

Also, the `purser-software` example also worked without a problem. Thanks!

FYI, and I mentioned this to @ryanchristo, in case you are going I will be at ETHDenver next week and will be participating in the hackathon. Currently without a team, and I am open to any number of ideas, but my interests are to use Colony within a social impact/charitable context.

**Comment by fuguefoundation** [comment]
I just tried my trezor again in my project and it worked! The only discernible difference from before was updating from Node 10.12 to 10.15. Hard to say exactly what the issue was. I did notice that when I was receiving the error before I was only prompted for my PIN and never subsequently for the passphrase (25th word), I'd just error out. But it seems to be working now so we'll chalk it up to a double win. Cheers!
